image_handler.py

```
@@ -0,0 +1,257 @@
+    image_urls = []
+    
+    if message.attachments:
+        logger.debug(f"Message has {len(message.attachments)} attachment(s)")
+        for attachment in message.attachments:
+            logger.debug(f"Attachment: {attachment.filename}, type: {attachment.content_type}")
+            # Check if attachment is an image
+            if attachment.content_type and attachment.content_type.startswith('image/'):
+                image_urls.append(attachment.url)
+                logger.info(f"Found image attachment: {attachment.filename} ({attachment.content_type})")
+    else:
+        logger.debug(f"Message has no attachments")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)